internal/app/app.go

```diff
@@ -1779,6 +1781,25 @@ func (app *App) repairCascadeNode(node *mysql.Node, clusterState map[string]*Nod
 	}
 }
 
+func (app *App) repairExternalReplication(masterNode *mysql.Node) {
```

Comment:
repairExternalReplication => TryRepairExternalReplication, for similarity with TryRepairReplication and maybe better move in separate file for external replication, like a common replication in replication.go

Reply:
No, this is an internal method, which is a wrapper for TryRepairReplication